airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/base_streams.py

```diff
@@ -48,7 +48,7 @@ def availability_strategy(self) -> Optional["AvailabilityStrategy"]:
     def __init__(self, api: "API", include_deleted: bool = False, page_size: int = 100, max_batch_size: int = 50, **kwargs):
         super().__init__(**kwargs)
         self._api = api
-        self.page_size = page_size if page_size is not None else 100
+        self.page_size = page_size if page_size is not None else 500
```

Comment:
thought: Since we have condition for handling `None` page_size I think we can remove this param from spec. As we want to make it default and not expose to user. wdyt?

Reply:
I think is better to follow the spec practice and define defaults in the specs. 
Even if we don't give a default the form will be invalid in the UI. I think it is better to force a default value we want than allow users to set one.